site/content/contribute/containers/_index.md

```diff
@@ -0,0 +1,56 @@
+
+- [mattermost/mattermost-team-edition](https://hub.docker.com/r/mattermost/mattermost-team-edition) - **Official Mattermost Server** image for the **Team Edition version**. To find the Dockerfile please refer to https://github.com/mattermost/mattermost-server/tree/master/build.
+
+- [mattermost/mattermost-push-proxy](https://hub.docker.com/r/mattermost/mattermost-push-proxy) - Mattermost Push Proxy. [Documentation](https://developers.mattermost.com/contribute/mobile/push-notifications/service/). [GitHub](https://github.com/mattermost/mattermost-push-proxy).
+
+- [mattermost/mattermost-loadtest](https://hub.docker.com/r/mattermost/mattermost-loadtest) - Image for the Load Test application. Tools for profiling Mattermost under heavy load. [GitHub](https://github.com/mattermost/mattermost-load-test).
+
+- [mattermost/mattermost-operator](https://hub.docker.com/r/mattermost/mattermost-operator) - Official image for Mattermost Operator for Kubernetes. For more information see https://github.com/mattermost/mattermost-operator.
+
+- [mattermost/mattermost-cloud](https://hub.docker.com/r/mattermost/mattermost-cloud) - Mattermost Private Cloud is a SaaS offering meant to smooth and accelerate the customer journey from trial to full adoption. For more information see https://github.com/mattermost/mattermost-cloud.
+
+## Community-Maintained Docker Images
+
+- [mattermost/mattermost-prod-app](https://hub.docker.com/r/mattermost/mattermost-prod-app) - Community driven image for Mattermost Server. **This Docker repository will be deprecated in Mattermost 6.0**. For more information and to check the Dockerfile please refer to https://github.com/mattermost/mattermost-docker.
+
+- [mattermost/mattermost-prod-db](https://hub.docker.com/r/mattermost/mattermost-prod-db) - Community driven image for Database to run together with **mattermost/mattermost-prod-app**. **This Docker repository will be deprecated in Mattermost 6.0**. For more information and to check the dockerfile please refer to https://github.com/mattermost/mattermost-docker
+
+- [mattermost/mattermost-prod-web](https://hub.docker.com/r/mattermost/mattermost-prod-web) - Community driven image for WebServer to run together with **mattermost/mattermost-prod-app**. **This Docker repository will be deprecated in Mattermost 6.0**. For more information and to check the dockerfile please refer to https://github.com/mattermost/mattermost-docker
+
+- [mattermost/mattermost-preview](https://hub.docker.com/r/mattermost/mattermost-preview) - This is a Docker image to install Mattermost in Preview Mode for exploring product functionality on a single machine using Docker. [Documentation](http://bit.ly/1W76riY). [GitHub](https://github.com/mattermost/mattermost-docker-preview).
```

Comment:
@cpanato Is this community-maintained?

Reply:
no this is our side, but we need to spend more time on that, if we want to continue doing that

Reply:
@cpanato okay, I moved under the Mattermost supported section